ably/rest/auth.py

```diff
@@ -170,79 +179,54 @@ def create_token_request(self, key_name=None, key_secret=None,
         if query_time is None:
             query_time = self.auth_options.query_time
 
-        if not token_params.get("timestamp"):
+        if not timestamp:
             if query_time:
-                token_params["timestamp"] = self.ably.time()
+                timestamp = self.ably.time()
             else:
-                token_params["timestamp"] = self._timestamp()
+                timestamp = self._timestamp()
+
+        token_request["timestamp"] = int(timestamp)
 
-        token_params["timestamp"] = int(token_params["timestamp"])
+        token_request['ttl'] = ttl or TokenDetails.DEFAULTS['ttl'] * 1000
```

Comment:
In both, python and ruby, when times are represented as an integer they are in seconds. This takes same approach as the ruby lib https://github.com/ably/ably-ruby/blob/0ba64081f1ee5117f3dafbb6e9fa59e026f471dc/lib/ably/auth.rb#L36

We can change it if you think it is wise.